### PR TITLE
(APS-356) Prepopulate catering

### PR DESCRIPTION
**Pull request by yndajas**
# Context

[JIRA](https://dsdmoj.atlassian.net/browse/APS-356)

# Changes in this PR

- Prepopulate the catering placement requirement on the matching information screen in Assess

## Screenshots of UI changes

(having a CSS problem locally so excuse the lofi screenshots)

### Before

image

### After

image

### CODE REVIEW

File: server/form-pages/utils/defaultMatchingInformationValues.test.ts [hunk 1]
```diff
@@ -49,11 +57,40 @@ describe('defaultMatchingInformationValues', () => {
...
   })
 
+  describe('isCatered', () => {
+    it('is set to the original value if defined', () => {
+      expect(
+        defaultMatchingInformationValues({ ...bodyWithUndefinedValues, isCatered: 'desirable' }, assessment),
+      ).toEqual(expect.objectContaining({ isCatered: 'desirable' }))
+    })
+
+    it("is set to 'essential' when there's no original value and `catering` === 'no'", () => {
```
Comment (edavey) [inline server/form-pages/utils/defaultMatchingInformationValues.test.ts:73]: thanks for laying it out clearly in the commit message that 'catering' equals "self-catering" but maybe for extra clarity we should reinforce that potential gotcha in our scenario title e.g

> is set to 'essential' when there's no original value and `catering` (self-catering) === 'no'

File: package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "lint:fix": "eslint . --cache --max-warnings 0 --fix && npm run shellcheck",
     "typecheck": "tsc && tsc -p integration_tests",
     "test": "jest --collectCoverage",
+    "test:debug": "jest --collectCoverage=false",
```
Comment (edavey) [inline package.json:26]: handy, thks, we have that in CAS2 also

**Review by edavey — APPROVED** [review]
Really clear, thank you.

I find that the new mocking is working well e.g.

```
when(retrieveQuestionResponseFromFormArtifact)
  .calledWith(assessment.application, Catering, 'catering')
  .mockReturnValue('no')
```

File: server/form-pages/utils/defaultMatchingInformationValues.test.ts (hunk 1 above)
Comment (yndajas) [reply server/form-pages/utils/defaultMatchingInformationValues.test.ts]: Good shout!

**Pull request merged**